runtime/src/gpu/nvidia/gpu-nvidia-reduce.cc

```diff
@@ -81,5 +81,30 @@ GPU_IMPL_REDUCE(DEF_ONE_REDUCE_RET_VAL_IDX, ArgMax, maxloc)
 
 #undef DEF_ONE_REDUCE_RET_VAL_IDX
 
+#define DEF_ONE_SORT(cub_kind, chpl_kind, data_type) \
```

Comment:
we should probably rename this file and its AMD counterpart. Again "cub" is OK to use in the name, these files are literally cub/hipcub wrappers.